test/index.html

```diff
@@ -10,7 +10,7 @@
   <div id="qunit"></div>
   <script src="../node_modules/qunitjs/qunit/qunit.js"></script>
 
-  <script src="../build/temp/ie8/videojs-ie8.min.js"></script>
+  <script src="../build/temp/ie8/videojs-ie8.js"></script>
```

Comment:
we probably should wrap this in the IE8 conditional, but that isn't necessary for this Pr.